README.md

```diff
@@ -32,14 +32,28 @@ on any other (or the same) supported protocol, for example to:
 
 [![Build Status](https://travis-ci.com/cbdevnet/midimonster.svg?branch=master)](https://travis-ci.com/cbdevnet/midimonster) [![Coverity Scan Build Status](https://scan.coverity.com/projects/15168/badge.svg)](https://scan.coverity.com/projects/15168)
 
+If you have any questions just write an Issue here on Github or get in touch with us via IRC on [Hackint](ircs://irc.hackint.org:6697) in `#midimonster`.
```

Comment:
Please replace `write` with `create` or `open`

It would probably be more useful to link to the webchat (see https://www.hackint.org/webchat)